backend/clubs/serializers.py

```diff
@@ -849,6 +850,23 @@ class Meta:
         fields = ("club", "person")
 
 
+class FavoriteTimeSerializer(serializers.ModelSerializer):
+    """
+    Used by users to get a list of clubs that they have favorited.
+    """
```

Comment:
I think I should have added an updated description for this method -- it's a new serializer because it also includes when the clubs were favorited. Also it's not for the user its for the club.